### PR TITLE
Always expanding the path given in the treefmt.toml

### LINKED ISSUE #102: Couldn't include a single file
This worked:

```toml
[formatter.cabal]
command = "cabal-fmt"
options = [
  "--inplace"
]
includes = ["*.cabal"]
```

This didn't

```toml
[formatter.cabal]
command = "cabal-fmt"
options = [
  "--inplace"
]
includes = ["scarf.cabal"]
```

though the debug output did say "1 literals" for cabal-fmt.

### PR DESCRIPTION
**Pull request by Rizary**
As discussed, since all path is absolute, then we should always expand the path in the `treefmt.toml`

 fixes #102 and #82

### CODE REVIEW

**Comment by zimbatm** [comment]
Can you add some tests?

It should show that:

```
[formatters.foo]
command = "echo"
includes = [ "foo" ]
```
Should match both a top-level "foo" file, and also a "foo" file in a sub-folder.

**Comment by Rizary** [comment]
It isn't included. Let me try again.

**Comment by zimbatm** [comment]
Add unit tests:

```
*.js - Should match any .js files in the work dir and below.
foo - Should match any file named "foo" in the work dir and below.
./foo - Should only match the file named "foo" in the work dir.
/foo - "
```

File: src/lib.rs [hunk 1]
```diff
@@ -57,6 +57,82 @@ pub fn expand_if_path(str: String, reference: &Path) -> String {
...
+
+        assert_eq!(path, test);
+    }
+    
+    #[test]
+    fn test_expand_if_path_single_pattern_with_slash() {
+        let path = vec![
+            "/Foo.hs",
+            "/nested/Foo.hs",
+            "/nested/nested_again/Foo.hs",
```
Comment (zimbatm) [inline src/lib.rs:121]:
```suggestion
            "/nested/nested/Foo.hs",
```

File: treefmt.toml [hunk 2]
```diff
@@ -23,8 +23,8 @@ options = [
     "--mode", "inplace",
     "--check-idempotence",
 ]
-includes = ["*.hs"]
-excludes = ["examples/haskell/"]
+includes = ["Foo.hs"]
+
```
Comment (zimbatm) [inline treefmt.toml:27]: revert
